R/drive_upload.R

```diff
@@ -66,35 +66,49 @@ drive_upload <- function(file = NULL,
   path <- path %||% root_folder()
   path <- as_dribble(path)
   confirm_single_file(path)
+  existing <- NULL
   if (!is_folder(path)) {
-    stop(
-      glue(
-        "Requested parent folder does not exist:\n{path$name}"
-      ),
-      call. = FALSE
-    )
+    if (!is.null(name)) {
+      stop(
+        glue(
+          "Requested parent folder does not exist:\n{path$name}"
+        ),
+        call. = FALSE
+      )
+    }
+    if (!overwrite) {
+      stop(
+        glue("File already exists:\n{path$name}"),
```

Comment:
Let's suggest the fix and bulletize the file as we are starting to do everywhere. In fact, that would be good for this entire PR.

> File exists. Use `overwrite = TRUE` to upload new content into this file id.